R/guess_dates.R

```diff
@@ -79,10 +81,13 @@ guess_dates <- function(x, error_tolerance = 0.1, first_date = NULL,
   ## well-formatted date, but still returning a character. If it can't convert
   ## it, it returns NA.
 
-  
+  # save the original x for later if nothing is converted
+  ox <- x
```

Comment:
I thought that variable was for real, but then realised it's just an `ox`. Fine.

Reply:
wait... is this a pun on hoax?